Sources/Core/Utility/Data+Extension.swift

```diff
@@ -41,6 +42,10 @@ extension Data {
     }
 
     public static func randomBytes(length: Int) -> Data? {
+        #if os(Linux)
+//        return Data(URandom().bytes(count: length)) 
+        return try? Data.random(length: length)
+        #else
```

Comment:
If this is necessary, why btw? But if it is, it's better to implement it in a way as backward capability implemented above.

Means implement exact same methods as existed one, but marked it with `@available(obsolete)` on all Apple platforms.

Reply:
The issue is SecRandomCopyBytes isn't available on Linux. I was trying to avoid spinning up a new process and use dev/random, but now believe it might be inevitable.